packages/clerk-js/src/core/clerk.ts

```diff
@@ -1654,6 +1672,16 @@ export class Clerk implements ClerkInterface {
     this.environment = environment;
   }
 
+  public prefetchDependencies = async (): Promise<void> => {
```

Comment:
I believe this can be private, and don't need to call it from `IsomorphicClerk`.